pkg/model/model.go

```diff
@@ -49,6 +49,13 @@ var (
 	}
 )
 
+// VulnerabilityLines is the representation of the found line for issue
+type VulnerabilityLines struct {
+	Line                 int
+	VulnLine             []VulnLines
```

Comment:
```suggestion
	VulnLines             []VulnLine
```

Reply:
I wonder if we should rename this internal struct to `[]CodeLine`

WDYT @felipe-avelar @joaoReigota1?

Reply:
Agree, will change to []CodeLine